File: src/main/kotlin/util/Command.java
package util;

public enum Command {
  START("!poker"),
  ROLL("!roll"),
  REROLL("!reroll"),
  PASS("!pass"),
  FINISH("!finish"),
  HELP("!help"),
  COMBINATION("!combo"),
  STATISTICS("!stat");

  public final String value;

  Command(String value) {
    this.value = value;
  }
}

File: src/main/kotlin/org/example/botfarm/util/Command.kt
package org.example.botfarm.util

/**
 * Enum representing different commands used in the poker game.
 *
 * @property value The string value of the command.
 */
enum class Command(val value: String) {
    START("poker"),
    ROLL("roll"),
    REROLL("reroll"),
    PASS("pass"),
    FINISH("finish"),
    HELP("help"),
    COMBINATION("combo"),
    STATISTICS("stat")
}
